### PR TITLE
A0-2301: Restyle the app to make it adapt to window resizing

### PR DESCRIPTION
**Pull request by jalooc**
Included changes:
* deleted the grey border in popup windows, as discussed with the designer;
* distinguished between the extension popup html file (index.html) and the "external" popup html file (external.html), since the extension popup has different rules related to sizes/resizing than the external popups;
* consolidated global (body) styles that were spread across different places;
* replaced hardcoded widths and max-widths with flexible styles.

### CODE REVIEW

File: packages/extension-ui/src/components/View.tsx [hunk 1]
```diff
@@ -49,6 +51,13 @@ export const BodyTheme = createGlobalStyle<ThemeProps>`
       display: none;
     }
   }
+
+  div#root {
+    height: calc(100vh - 2px);
+    margin: 0 auto;
+    max-width: 100%;
+    padding: 0;
+  }
```
Comment (youPickItUp) [inline packages/extension-ui/src/components/View.tsx:60]: I think giving html and body 100% allows us to give every child 100% without worrying about +/- px :thinking:

File: packages/extension-ui/src/components/Table.tsx [hunk 2]
```diff
@@ -23,8 +23,9 @@ function Table({ children, className = '', isFull }: Props): React.ReactElement<
 export default React.memo(
   styled(Table)(
     ({ theme }: ThemeProps) => `
+  display: table;
+  table-layout: fixed;
```
Comment (youPickItUp) [inline packages/extension-ui/src/components/Table.tsx:27]: hahaha, omg, YES!
btw. isn't display: table a default?

File: packages/extension-base/src/background/handlers/Extension.ts [hunk 3]
```diff
@@ -477,7 +477,7 @@ export default class Extension {
 
   // this method is called when we want to open up the popup from the ui
   private windowOpen (path: AllowedPath): boolean {
-    const url = `${chrome.extension.getURL('index.html')}#${path}`;
+    const url = `${chrome.extension.getURL('external.html')}#${path}`;
```
Comment (youPickItUp) [inline packages/extension-base/src/background/handlers/Extension.ts:480]: I thought `notification.html` is new window popup and `index.html` is an extension popup, isn't that correct?

**Review by youPickItUp — APPROVED** [review]

**Comment by youPickItUp** [comment]
There is still a border around signed / declined views:
is that expected?
image

File: packages/extension-base/src/background/handlers/Extension.ts (hunk 3 above)
Comment (jalooc) [reply packages/extension-base/src/background/handlers/Extension.ts:480]: It wasn't like that - `index.html` was used for both extension popup and _some_ external windows. What I did was splitting this use case into 2 separate html files. So now we have one _isolated_ usage of the extension popup (`index.html`) and 2 types of external windows (the new `external.html` and the old, intouched `notification.html`).

Maybe I could've merged the 2 types of external windows, but the previous division acros _different axis of responsibilities_ makes me uncomfortable to do so without vast manualt testing that'd take much more time than this ticket deserves.

File: packages/extension-ui/src/components/Table.tsx (hunk 2 above)
Comment (jalooc) [reply packages/extension-ui/src/components/Table.tsx:27]: Oh yeah, sure 👍

File: packages/extension-ui/src/components/View.tsx (hunk 1 above)
Comment (jalooc) [reply packages/extension-ui/src/components/View.tsx:60]: What are you suggesting exactly?

**Comment by jalooc** [comment]
> There is still a border around signed / declined views:
> is that expected?

Haha, it's visually the same things as the grey ones, but the colorful ones are realised using `outline`, as opposed to the grey one done with an svg.

But yeah, we don't want that according to the new designs 👍

File: packages/extension-ui/src/components/View.tsx (hunk 1 above)
Comment (youPickItUp) [reply packages/extension-ui/src/components/View.tsx:60]:
```
html {
  height: 100%;
}
body {
  height: 100%;
}
div#root {
  height: 100%;
}

Comment (jalooc) [reply packages/extension-ui/src/components/View.tsx:60]: Yeah, I left these styles _as is_, but I think we can do that 👍

**Pull request merged**